README.md

```diff
@@ -38,6 +39,11 @@ To run tests locally, run `make test`. Please don't push to master with
 failing tests—Jenkins will refuse to deploy your code, and nobody will be able
 to deploy until fixing it.
 
+If you make a pull request to the OCF GitHub organization from your fork of
+ocfweb, Jenkins will attempt to build your branch, and if your build fails,
+you can log into [Jenkins](https://jenkins.ocf.berkeley.edu) with your OCF
```

Comment:
I don't think logging in here helps at all, the same results are shown, so just going to Jenkins should be sufficient.